README.md

```diff
@@ -229,6 +230,39 @@ In order to run the boxel-motion demo app:
 2. `pnpm start`
 3. Visit http://localhost:4200 in your browser
 
+## Payment Setup
+
+There is some pre-setup needed to enable free plan on development account:
+
+1. Use stripe cli to listen the webhook
+
+```
+stripe listen --forward-to localhost:4201/_stripe-webhook --api-key sk_test_api_key_from_the_sandbox_account
+```
+
+2. You will get webhook signing secret from stripe cli after Step 1 is done
+
+```
+> Ready! You are using Stripe API Version [x]. Your webhook signing secret is whsec_xxxxx
+```
+
+3. Set STRIPE_WEBHOOK_SECRET environment value in `packages/billing/stripe-webhook-handlers/index.ts` file
+
+```
+process.env.STRIPE_WEBHOOK_SECRET =
```

Comment:
Setting environment from within the app doesn't sound like a right thing to do. Instead, pass the env variables to the realm server app as it starts:

`STRIPE_WEBHOOK_SECRET=... STRIPE_API_KEY=... pnpm start:all`

We have a ticket to add a `.env.development` file that will contain these variables but for now they need to be passed in in an inline fashion